samples/config_helper_vulkan.cc

```diff
@@ -1070,6 +1122,12 @@ void ConfigHelperVulkan::DumpPhysicalDeviceInfo() {
   if (vkGetPhysicalDeviceProperties2KHR) {
     std::cout << "  driverName: " << driver_properties.driverName << std::endl;
     std::cout << "  driverInfo: " << driver_properties.driverInfo << std::endl;
+    if (supports_subgroup_size_control_) {
+        std::cout << "  minSubgroupSize: " << subgroup_size_control_properties.minSubgroupSize << std::endl;
```

Comment:
These seem longer then 80 chars, there is a `./tools/format` script which will run `clang-format` over all the files.

Reply:
I run `clang-format`, should look ok now.